Purchases/Purchasing/PurchasesOrchestrator.swift

```diff
@@ -304,6 +339,13 @@ private extension PurchasesOrchestrator {
 
 }
 
+extension PurchasesOrchestrator: StoreKit2TransactionListenerDelegate {
+
+    func transactionsUpdated() {
+        syncPurchases(receiptRefreshPolicy: .always, isRestore: false, maybeCompletion: nil)
```

Comment:
I'm probably not familiar enough to know this answer, but anytime `finish(transaction: StoreKit.Transaction)` is done, this delegate gets called. I would have though that it would be called for restoring as well. Is that not the case?

Reply:
I'm not quite sure what you mean, could you clarify? 
the intent of the delegate was for `SK2TransactionListener` to have a way to notify other classes that a change in transactions was detected, so that we can re-post the receipt. 
When a change is detected, we post the SK1 receipt using `syncPurchases`. The `isRestore` parameter here is used by the backend, and it not even be relevant, I'll add a `todo` to double-check.